admin/includes/languages/english.php

```diff
@@ -698,5 +698,4 @@ function zen_date_raw($date, $reverse = false) {
 ///////////////////////////////////////////////////////////
 // include additional files:
   require(DIR_WS_LANGUAGES . $_SESSION['language'] . "/" . FILENAME_EMAIL_EXTRAS);
-  include(zen_get_file_directory(DIR_FS_CATALOG_LANGUAGES . $_SESSION['language'] . '/', FILENAME_OTHER_IMAGES_NAMES, 'false'));
-  
\ No newline at end of file
+  include(zen_get_file_directory(DIR_FS_CATALOG_LANGUAGES . $_SESSION['language'] . '/', FILENAME_OTHER_IMAGES_NAMES));
```

Comment:
I thought that somewhere in "here" (GitHub) that I had commented/asked about this use of the default value in the call to the function and had received a response indicating that it was desirable to maintain the `'false'` statement... I can't seem to find that (or perhaps I'm thinking of another similar usage). So I ask what is the background in removing it from here? Further, is a similar change going to be pushed (pulled? :)) to all such uses?

Reply:
No concrete reason, just as it is the default.